sdk/aqueduct/integrations/dynamic_k8s_integration.py

```diff
@@ -34,16 +54,49 @@ def create(self) -> None:
             )
 
         status = engine_statuses[self._metadata.name].status
-        if status == K8sClusterStatusType.ACTIVE:
+        if status == K8sClusterStatusType.ACTIVE and all(
+            value is None for value in config_delta.dict().values()
+        ):
             print("Cluster is already in %s status." % status.value)
             return
 
         print(
-            "Cluster is currently in %s status. It could take 12-15 minutes for the cluster to be ready..."
+            "Cluster is currently in %s status. It could take 12-15 minutes for the cluster to be created or updated..."
+            % status.value
+        )
+        globals.__GLOBAL_API_CLIENT__.edit_dynamic_engine(
+            action="create",
+            integration_id=str(self._metadata.id),
+            config_delta=config_delta,
+        )
+
+    def update(self, config_delta: Union[Dict[str, str], DynamicK8sConfig] = {}) -> None:
+        config_delta = parse_dynamic_k8s_config(config_delta)
+
+        engine_statuses = globals.__GLOBAL_API_CLIENT__.get_dynamic_engine_status(
+            engine_integration_ids=[str(self._metadata.id)]
+        )
+        if len(engine_statuses) != 1:
```

Comment:
nit: I'd find it a bit more readable if we change this to `len(engine_statuses) == 0` and assert `len(engine_status) <= 1`. Adds extra protection too against unexpected duplicate integrations with the same name.